src/util/geodetic.cpp

```diff
@@ -103,6 +103,59 @@ glm::dvec3 geoPositionFromCamera() {
     return glm::dvec3(lat, lon, altitude);
 }
 
+glm::dvec2 geoViewFromCamera() {
+    const SceneGraphNode* n = global::navigationHandler->orbitalNavigator().anchorNode();
+    if (!n) {
+        return glm::dvec3(0.0);
+    }
+    const Renderable* renderable = n->renderable();
+    if (!renderable) {
+        return glm::dvec3(0.0);
+    }
+
+    const glm::dvec3 cameraPosition = global::navigationHandler->camera()->positionVec3();
+    const glm::dvec3 cameraViewDirection =
+        global::navigationHandler->camera()->viewDirectionWorldSpace();
+    const glm::dmat4 inverseModelTransform = glm::inverse(n->modelTransform());
+
+    const glm::dvec3 cameraPositionModelSpace = glm::dvec3(inverseModelTransform *
+        glm::dvec4(cameraPosition, 1.0)
+    );
```

Comment:
```suggestion
    const glm::dvec3 cameraPositionModelSpace =
        glm::dvec3(inverseModelTransform * glm::dvec4(cameraPosition, 1.0));
```

Bit cleaner to see how the calculation happens